submittable/react-select
# Fixed input width sizing

If you typed a really long string into the input it would break the layout. This fixes that
https://github.com/JedWatson/react-select/issues/1127

Test
_____
* Run `npm start`
* Goto http://localhost:8000/
* Type something long into an input, it should not break the layout
![](http://g.recordit.co/XlZvkD0nU6.gif)
